README.md

```diff
@@ -89,6 +89,19 @@ mvn verify -Proad-traffic-demo
 python -m unittest discover
 ```
 
+## Building the documentation
+
+To build the docs locally, assuming you have Python installed, install the docs dependencies:
+```bash
+cd docs/
+pip install -r requirements.txt
+```
+
+Then use the sphinx Makefile:

```

Comment:
There's also a Windows `.bat`, do we want to mention that?

Reply:
I am not sure that's required really